/home/ubuntu/Documents/source/callback-verification/cbverifier/driver.py:71: DeprecationWarning: BaseException.message has been deprecated as of Python 2.6
  e.message))
Traceback (most recent call last):
  File "/home/ubuntu/Documents/source/callback-verification/cbverifier/driver.py", line 502, in <module>
    main()
  File "/home/ubuntu/Documents/source/callback-verification/cbverifier/driver.py", line 409, in main
    driver = Driver(driver_opts)
  File "/home/ubuntu/Documents/source/callback-verification/cbverifier/driver.py", line 71, in __init__
    e.message))
Exception: An error happened reading the trace in /home/ubuntu/Documents/data/monkey_traces/alert-dialog-traces-1/_traces/Wizcorp-phonegap-plugin-localNotifications-platforms/monkeyTraces/trace-HelloCordova-debug_2017-04-15_20:19:50.repaired (Wrong integer value in message!)
real 0.56
user 0.46
sys 0.05
